diff --git a/example/test_televuer.py b/example/test_televuer.py
index b9111b8..34c95d1 100644
--- a/example/test_televuer.py
+++ b/example/test_televuer.py
@@ -14,7 +14,7 @@ def run_test_TeleVuer():
     use_hand_track = False
     # teleimager, if you want to test real image streaming, make sure teleimager server is running
     from teleimager.image_client import ImageClient
-    img_client = ImageClient(host="192.168.123.164")
+    img_client = ImageClient(host="192.168.0.109", request_bgr=True)
     camera_config = img_client.get_cam_config()
     # teleimager + televuer
     tv = TeleVuer(use_hand_tracking=use_hand_track, 
@@ -24,7 +24,7 @@ def run_test_TeleVuer():
                   display_mode="immersive",   # "ego" or "immersive" or "pass-through"
                   zmq=camera_config['head_camera']['enable_zmq'],
                   webrtc=camera_config['head_camera']['enable_webrtc'],
-                  webrtc_url=f"https://192.168.123.164:{camera_config['head_camera']['webrtc_port']}/offer"
+                  webrtc_url=f"https://192.168.0.109:{camera_config['head_camera']['webrtc_port']}/offer"
                   )
     # pure televuer
     # tv = TeleVuer(use_hand_tracking=use_hand_track, 
@@ -41,7 +41,12 @@ def run_test_TeleVuer():
         input("Press Enter to start TeleVuer test...")
         running = True
         while running:
-            img, _= img_client.get_head_frame()
+            tele_img = img_client.get_head_frame()
+            img = tele_img.bgr
+            if img is None:
+                logger_mp.warning("Head camera frame is None, skipping render...")
+                time.sleep(0.016)
+                continue
             tv.render_to_xr(img)
 
             start_time = time.time()
